.github/workflows/goldsky-deploy.yml

```
@@ -0,0 +1,36 @@
+      - ".github/workflows/goldsky-deploy.yml"
+  workflow_dispatch:
+
+jobs:
+  deploy-goldsky:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)